ui/src/utils/helpers.jsx

```diff
@@ -29,3 +30,24 @@ export function getStatusChip(status) {
 
   return <span className={`${bgColor} text-white py-1 px-2 rounded-full text-xs`}>{status}</span>;
 }
+
+export function extractOrderDetail(order, getDisplayInfo) {
+  const orderType = Object.keys(order)[0];
+  const { value, brand } = order[orderType];
+  const { assetKind, decimalPlaces } = getDisplayInfo(brand);
+
+  const valueString = stringifyValue(value, assetKind, decimalPlaces);
+  const currency = orderType === 'Price' ? 'IST' : 'BLD';
```

Comment:
We should use the issuerNames coming from the `store.getIssuerName` instead of the hard coded ones